mlibrary/search
# Removing `src/modules/analytics`.

# Overview
`GAListener` was not used in the site, so it has been deleted. When working on simplifying `handleGAClick`, it was realized that it would never call because no element exists with the attributes starting with `data-ga-`. The function was removed, thus making `src/modules/analytics` empty.

## Anything else?
### `ReactGA`
`react-ga` was initializing a retired Universal Analytics account, making it useless. The code has been removed. It was the only instance of using `react-ga`, so the dependency has been uninstalled.

### Pull request templates
With uninstalling `react-ga`, the pull request templates have been updated to list the current set of dependencies, along with more accurate information as to why the pull request exists.

### `App`
To align with [current React practices](https://react.dev/learn/your-first-component), this class component has been converted to a functional component:
- `App`

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Load the site and poke around to see if anything is broken.
